tests/Fixer/ArrayNotation/YieldFromArrayToYieldsFixerTest.php

```diff
@@ -243,5 +243,48 @@ function foo() {
                 yield from [1, 2, 3];
             }',
         ];
+
+        yield 'from empty arrays' => [
```

Comment:
I believe data set's name should represent the intent of the test case.

```suggestion
        yield 'skip empty arrays' => [
```